### DISCUSSION TITLE
Is Ender 3 KE supported by 3MS?

### DISCUSSION THREAD

**Discussion by RoyMessin** [3DCoded/3MS#33] [General Help]
It runs a Klipper version on its own board.

**Comment by 3DCoded** [comment]
The 3MS currently has the (highy recommended) option of using Happy Hare firmware. Happy Hare does support the K1 series. I'm not sure about the Ender 3 KE though. I would recommend asking this on [Happy Hare](https://github.com/moggieuk/Happy-Hare)'s Github.
